sdk/core/core-sse/test/internal/createStream.spec.ts

```diff
@@ -32,17 +36,6 @@ describe("createStream", () => {
     assert.isTrue(canceled);
   });
 
-  it("creates disposable stream", async function (this: Context) {
```

Comment:
Is this no longer valid, how come?

Reply:
Reverted test and used the dispose manually instead of using the `using` statement